docs/OV_Runtime_UG/deployment/conditional_compilation_deployment.md

```diff
@@ -0,0 +1,289 @@
+# OpenVINO conditional compilation for optimal binary size{#opevino_conditional_compilation_deploying_guide}
```

Comment:
Why do we need to include this documentation on the site? Maybe we can add the reference to this documentation:
https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/conditional_compilation.md

We also don't have documentation on the site how to build OV and it is ok. I think here we have the same approach. Maybe we can have some blog post to demonstrate this functionality.

In case if anyway we would like to have this information on the site, I suppose to reuse the information from the [page](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/conditional_compilation.md)

Reply:
@ilyachur  this requirement is from https://jira.devtools.intel.com/browse/CVS-103482: we need a tutorial for CC deployment, which should be have different purpose with https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/conditional_compilation.md.

Reply:
As the discussion, this PR will be for conditional compilation users. While [docs/dev/conditional_compilation.md|https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/conditional_compilation.md] is for contributors.